api/Cargo.toml

```diff
@@ -34,8 +34,8 @@ diem-types = { path = "../types" }
 diem-workspace-hack = { version = "0.1", path = "../crates/diem-workspace-hack" }
 diem-api-types = { path = "./types", package = "diem-api-types" }
 storage-interface = { path = "../storage/storage-interface" }
-move-core-types = { git = "https://github.com/0LNetworkCommunity/move-0L", rev = "94e1039c9fdf1472c2c7775de43135d28dafc955" }
-move-resource-viewer = { git = "https://github.com/0LNetworkCommunity/move-0L", rev = "94e1039c9fdf1472c2c7775de43135d28dafc955" }
+move-core-types = { git = "https://github.com/0LNetworkCommunity/move-0L", branch = "v6" }
```

Comment:
We want to use rev instead of branch. Using branch is discouraged since it requires manual `cargo update` which is super easy to forget, causes inconsistency and confusion. Plus, our upstreams Diem, Aptos are also using rev more than branches.

Reply:
Interesting, I was under the impression that this upgrade aims to remove rev and instead use `branch` for easier update in the future. Any concern about build reproducibility can be addressed by a static `move.lock` with own rev.
I understand your concern about manual `cargo update` but if someone want to update the rev they would have to rerun `cargo update` anyway.

Reply:
No, you do not need `cargo update` if you use `rev`, you can test it. That's why rev is suggested in the community.